ui/daml/UI/Onboarding.daml

```diff
@@ -100,137 +86,66 @@ handleOnboardingInstruction operator customer mp ins = case ins of
         void $ exercise clearingOfferCid Market.Accept
 
     ot@OnboardAuction{..} -> do
+        let account = Custody.createAccount custodian customer
         custodyCid <- createOrLookup Custody.Service with provider = custodian; ..
 
-        tradingAccount    <- createOrLookupAccount    ot custodyCid optTradingAccount "Auction-TradingAccount"
-        receivableAccount <- createOrLookupAccount    ot custodyCid optReceivableAccount "Auction-ReceivableAccount"
-        allocationAccount <- createAllocationAccount' ot custodyCid "allocation"
-
         offerCid <- exerciseByKey @Distributor.Role (operator, provider) Distributor.OfferAuctionService with ..
-        void $ exercise offerCid Auction.Accept with ..
+        void $ exercise offerCid Auction.Accept
 
     ot@OnboardBidding{..} -> do
+        let account = Custody.createAccount custodian customer
         custodyCid <- createOrLookup Custody.Service with provider = custodian; ..
 
```

Comment:
Same question as above: is creating a custody service needed here, now that we have removed the account dependency?

Reply:
I don't think its needed anymore as it adds confusion (especially in the UI). I'll remove it but it does require a bit of effort to do so on the UI side.